docs/getting-started/developers/index.md

```diff
@@ -51,22 +65,43 @@ export default defineConfig({
 });
 ```
 
-#### Adding Uno to your build tool
+#### Add Uno to your build tool
```

Comment:
Maybe we should align with how the setup was explained in the [css docs](https://github.com/warp-ds/css-docs/pull/59/files). There we also suggested not to include uno in the build step after all due to possible issues unocss may have when used with VSCode plugins for autocompletion. Those will work best if a `uno.config.js` file is found according to their documentation.